src/go/debug-collector/README.md

```diff
@@ -0,0 +1,6 @@
+#Debug collector tool
+
+Collects all necessary data for debugging from a cluster and creates a tar file with it
+
+Installed and configured kubectl is needed  for work
+
```

Comment:
1. From what cluster (what type of cluster)? Buy the kubectl requirement one can guess that it's a k8s/opeshift cluster, but anyway...
2. What necessary data? Pods logs, resources etc.
3. s/tar file/tar.gz archive. Also, write down here what filename is (and where to look for it)
4. Describe how to run (flags etc)

Keep in mind this readme is for the users who don't have to look in code to figure out what is this for and how to use it.